### PR TITLE
7 build desktop homepage

### LINKED ISSUE #7: Build desktop homepage
This should include:
* header text
* two CTAs for hiring/looking for a new role with buttons + links leading to different actions/pages (need to confirm where learn more links are going)
* job seekers call-out
* employers call out

figma here: https://www.figma.com/file/BUUz9uQoqaOda6AK3dNmEh/Riveter-Trade?type=design&node-id=122-2&mode=design&t=lKZVm7bEPrLE4FhG-0

### PR DESCRIPTION
**Pull request by cleverettgirl**

### CODE REVIEW

File: app/components/SUSI/SIgnUpButton.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Button} from '@mui/material'
+
+
+export const SignUpButton = () => {
+	return (
+			<Button sx={{padding: '12px 52px', backgroundColor: '#0265DC', color: '#FFF', marginTop: '26px', marginBottom: '16px'}}>SIGN UP</Button>
+	)
+}
```
Comment (deejkelms) [inline app/components/SUSI/SIgnUpButton.tsx]: do you have eslint or prettier installed? it should autoformat this

File: app/components/nav/LearnMoreLink.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import Link from 'next/link'
+import ChevronRightIcon from '@mui/icons-material/ChevronRight'
+
+export const LearnMoreLink = () => {
+	return (
+		<Box sx={{color: '#0265DC', fontWeight: 500, letterSpacing: '.2px', display: 'flex', alignItems: 'center', padding: '8px 0'}}>
+			<Link href="#" style={{display: 'flex', alignItems: 'center'}}>
+				LEARN MORE
+				<ChevronRightIcon fill="#0265DC" />
+			</Link>
```
Comment (deejkelms) [inline app/components/nav/LearnMoreLink.tsx:12]: for this I think we can just use button? that might help cut down on some of these styles. https://mui.com/material-ui/react-button/#buttons-with-icons-and-label

File: app/components/ui/Divider.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { Box } from "@mui/material"
+
+export const Divider = ({color = "#000"}: {color?: string}) => {
+  return (
+    <Box height="1px" sx={{backgroundColor: color}} />
+  )
```
Comment (deejkelms) [inline app/components/ui/Divider.tsx:6]: material also has this? https://mui.com/material-ui/react-divider/

File: app/components/SUSI/SIgnUpButton.tsx (hunk 1 above)
Comment (cleverettgirl) [reply app/components/SUSI/SIgnUpButton.tsx]: installed

File: app/components/nav/LearnMoreLink.tsx (hunk 2 above)
Comment (cleverettgirl) [reply app/components/nav/LearnMoreLink.tsx:12]: I can add a TODO to fix this is a future branch or ticket

File: app/components/ui/Divider.tsx (hunk 3 above)
Comment (cleverettgirl) [reply app/components/ui/Divider.tsx:6]: same

**Pull request merged**